src/reconcile/rec_write.c

```diff
@@ -374,6 +374,22 @@ __rec_write_page_status(WT_SESSION_IMPL *session, WT_RECONCILE *r)
     page = r->page;
     mod = page->modify;
 
+    /*
```

Comment:
Upon successful reconciliation of a page, we track the max transaction id and timestamp irrespective of whether the page is being kept dirty or not?